app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -101,9 +101,17 @@ class SyncIndicatorViewController @Inject constructor(
   }
 
   private fun startSync(events: Observable<UiEvent>): Observable<UiChange> {
+    fun errorsStream() = dataSync
+        .streamSyncErrors()
+        .take(1)
+        .map { { ui: Ui -> ui.showErrorDialog(it) } }
+
+    val syncStream = dataSync
+        .sync(null)
```

Comment:
We should not trigger a sync if a sync is already ongoing. This will trigger a new sync every time a click happens, right?

Reply:
Done.